app/src/main/java/ru/anisart/vv/MainActivity.kt

```diff
@@ -33,17 +33,17 @@ import java.util.*
 @RuntimePermissions
```

Comment:
**Reporter**: ANDROIDLINT
**Rule**: CallNeedsPermission
**Severity**: ERROR
**File**: C:/Users/aanisimov/.jenkins/workspace/vv/app/src/main/java/ru/anisart/vv/MainActivity.kt L122

Call the corresponding &quot;WithPermissionCheck&quot; method of the generated PermissionsDispatcher class instead&#10;Trying to access permission-protected method directly&#10;Directly invoking a method annotated with @NeedsPermission may lead to misleading behaviour on devices running Marshmallow and up. Therefore, it is advised to use the generated PermissionsDispatcher class instead, which provides a &quot;WithPermissionCheck&quot; method that safely handles runtime permissions.


*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-1958654591>*